exporter/collector/googlemanagedprometheus/monitoredresource.go

```diff
@@ -94,12 +120,6 @@ func getStringOrDefault(attributes pcommon.Map, orElse string, keys ...string) s
 			return val.Str()
 		}
 	}
-	if contains(keys, string(semconv.AttributeServiceName)) {
```

Comment:
I think this is supposed to stay, right?

Reply:
yup my mistake